Source/CarthageKitTests/ResolverSpec.swift

```diff
@@ -84,11 +84,11 @@ class ResolverSpec: QuickSpec {
 
 			var generator = dependencies.generate()
 
-			// Dependencies should be listed in build order.
+			// Dependencies should be listed in lexical order of its project name.
```

Comment:
We should make sure we add new tests that the dependencies in `Cartfile.resolved` are sorted in build order.